app/controllers/idv/unavailable_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Idv
+  class UnavailableController < ApplicationController
+    def show
+      if FeatureManagement.idv_available?
+        if from_create_account?
+          return redirect_to sign_up_email_url
```

Comment:
I think you can move the `render` call directly into this controller if you handled these redirects in a `before_action`. A `redirect_to` call in a before action will break the before action chain.

Reply:
TY! I moved a few things around in this controller and eliminated the explicit `render` call based on @aduth's feedback.

Reply:
I think what @jmhooper was suggesting was to move the redirect behavior into a separate method to be called as a `before_action`, something like...

```rb
    before_action :redirect_if_idv_available_and_from_create_account

    def show
      analytics.vendor_outage(
        vendor_status: {
          acuant: IdentityConfig.store.vendor_status_acuant,
          lexisnexis_instant_verify: IdentityConfig.store.vendor_status_lexisnexis_instant_verify,
          lexisnexis_trueid: IdentityConfig.store.vendor_status_lexisnexis_trueid,
          sms: IdentityConfig.store.vendor_status_sms,
          voice: IdentityConfig.store.vendor_status_voice,
        },
        redirect_from: from,
      )
    end

    private

    def redirect_if_idv_available_and_from_create_account
      redirect_to sign_up_email_url if FeatureManagement.idv_available? && from_create_account?
    end
```

Reply:
Yeah, that is exactly what I was suggesting